console/src/test/java/org/web3j/console/project/TemplatesAsString.java

```diff
@@ -0,0 +1,370 @@
+/*
+ * Copyright 2019 Web3 Labs LTD.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.console.project;
+
+public class TemplatesAsString {
```

Comment:
Everything looked pretty good until I reached this class. Please write a method that will read this from templates in the resources and populate accordingly. It is ok to duplicate the resources and add a `.test` at the end of the test resources. Remember the tests are not packaged in the final jar.